<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Action Recognition</string>
  </property>
  <widget class="QLabel" name="VideoLabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>130</y>
     <width>731</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>Client Id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>Emp Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>Mobile No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>Emp Id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>open_black.png</normaloff>open_black.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>538</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>PROCESSING</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProcessBar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="WebcamButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>540</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Playout</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Copyright.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>Submit_Button.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClientId">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>e.g 001</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmpName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>26</number>
   </property>
   <property name="placeholderText">
    <string>e.g John</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmpMobile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>e.g 9874563210</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmpId">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>e.g 002</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Notification">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
